go_dependencies.bzl

```diff
@@ -4,9 +4,8 @@ def bb_storage_go_dependencies():
     go_repository(
         name = "com_github_aws_aws_sdk_go",
         importpath = "github.com/aws/aws-sdk-go",
-        sha256 = "6ba3169493880a63128b6c6edc9119817df257db0b34b27887cad871767f0525",
-        strip_prefix = "aws-sdk-go-1.16.26",
-        urls = ["https://github.com/aws/aws-sdk-go/archive/v1.16.26.tar.gz"],
+        sum = "h1:eydMoSwfrSTD9PWKUJOiDL7+/UwDW8AjInUGVE5Llh4=",
+        version = "v1.33.12",
     )
 
```

Comment:
Tiny nit: the existing `go_repository()` declarations have a newline between them, while the newly added ones don't. Be sure to remove the empty lines from the existing ones.

(Is there any way we can sort all of them alphabetically, by the way?)

Reply:
Done, no easy way to sort them though...